src/App.js

```diff
@@ -1,5 +1,98 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+
 class App {
-  async run() {}
+  async getCarNames(query) {
+    try {
+      const input = await Console.readLineAsync(query);
+      const carNames = input.split(',').map((name) => name.trim());
+      return carNames;
+    } catch (error) {
+      throw new Error(`[ERROR] ${error}`);
+    }
+  }
+
+  async getTryNumber(query) {
+    try {
+      const input = await Console.readLineAsync(query);
+      if (Number.isNaN(Number(input))) {
+        throw new Error(`[ERROR]`);
+      }
+      return input;
+    } catch (error) {
+      throw new Error(`[ERROR] ${error}`);
+    }
+  }
+
+  // 입력받은 경주용 자동차 이름을 {이름: 점수}의 점수판 객체로 만들기
+  setGame(input) {
+    const scoreBoard = {};
+    input.forEach((name) => {
+      scoreBoard[name] = 0;
+    });
+    return scoreBoard;
+  }
+
+  // 1회차 실행결과 계산하기
+  executionResult(input) {
+    const updatedInput = { ...input };
+
+    Object.keys(updatedInput).forEach((item) => {
+      const isFowrad = Random.pickNumberInRange(0, 9);
+      if (isFowrad >= 4) {
+        updatedInput[item] += 1;
+      }
+    });
+    return updatedInput;
+  }
+
+  printResult(input) {
+    const inputKeys = Object.keys(input);
+    inputKeys.forEach((item) => {
+      Console.print(`${item} : ${'-'.repeat(input[item])}`);
+    });
+  }
+
+  findWinnder(input) {
+    const inputKeys = Object.keys(input);
+    const inputValues = Object.values(input);
+
+    const highScore = Math.max.apply(null, inputValues);
+
+    const winners = [];
+
+    inputKeys.forEach((item) => {
+      if (input[item] === highScore) {
+        winners.push(item);
+      }
+    });
+
+    return winners;
+  }
+
+  printWinner(input) {
+    const winnersString = input.join(', ');
+    Console.print(`최종 우승자 : ${winnersString}`);
```

Comment:
`winnerNames`와 같은 변수명은 어떨까요? 
`String`이라는 타입을 적지 않고도, 어떤 의미가 담긴 변수인지 변수명으로도 충분히 나태낼 수 있을 것 같습니다 :)

```suggestion
    Console.print(`최종 우승자 : ${winnersNames}`);
```

Reply:
좋은 지적이라고 생각합니다! 변수명 짓는게 가장 어려워요 ㅎㅎ;;